Buildings/Controls/OBC/ASHRAE/G36_PR1/TerminalUnits/ZoneModeAndSetPoints.mo

```diff
@@ -0,0 +1,366 @@
+within Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits;
+block ZoneModeAndSetPoints "Output zone setpoint with operation mode selection"
+
+  parameter Integer numZon(min=2)
+    "Total number of served zones/VAV boxes";
+  parameter Modelica.SIunits.Temperature THeaOn=293.15
+    "Heating setpoint during on";
+  parameter Modelica.SIunits.Temperature THeaOff=285.15
+    "Heating setpoint during off";
+  parameter Modelica.SIunits.Temperature TCooOn=297.15
+    "Cooling setpoint during on";
+  parameter Modelica.SIunits.Temperature TCooOff=303.15
+    "Cooling setpoint during off";
+  parameter Modelica.SIunits.Time preWarCooTim=10800
+    "Maximum cool-down/warm-up time"
+    annotation (Evaluate=true, Dialog(tab="Operation mode", group="Parameters"));
+  parameter Modelica.SIunits.TemperatureDifference bouLim=1.1
+    "Value limit to indicate the end of setback/setup mode"
+    annotation (Evaluate=true, Dialog(tab="Operation mode", group="Parameters"));
+  parameter Modelica.SIunits.Temperature freProThrVal=277.55
+    "Threshold zone temperature value to activate freeze protection mode"
+    annotation (Evaluate=true, Dialog(tab="Operation mode", group="Parameters"));
+  parameter Modelica.SIunits.Temperature freProEndVal=280.35
+    "Threshold zone temperature value to finish the freeze protection mode"
+    annotation (Evaluate=true, Dialog(tab="Operation mode", group="Parameters"));
+  parameter Modelica.SIunits.Time warCooTim=1800
+    "Defined cool-down/warm-up time"
+    annotation (Evaluate=true, Dialog(tab="Operation mode", group="Test setting"));
+  parameter Boolean have_occSen=false
+    "Check if the zone has occupancy sensor"
+    annotation (Evaluate=true, Dialog(tab="Setpoint adjust", group="Sensors"));
+  parameter Boolean have_winSen=false
+    "Check if the zone has window status sensor"
+    annotation (Evaluate=true, Dialog(tab="Setpoint adjust", group="Sensors"));
+  parameter Boolean cooAdj=false
+    "Flag, set to true if both cooling and heating setpoint are adjustable separately"
```

Comment:
I don't think one needs to say that this is a flag. Same goes for the following few.